CHANGELOG

```diff
@@ -10,6 +10,9 @@ Current head v.2.1.0 rc
   * Remove first server impl (#16)
 - CMake
   * Remove dead code (#15)
+  * Removed first server impl (#16)
```

Comment:
Duplicated line (see line 10)

Reply:
Oups, I didn't pay attention when managing the conflict. Resolved below.